README.md

```diff
@@ -135,7 +135,7 @@ This architecture might seem like an overkill for a counter app, but the beauty
 
 So, what are you waiting for?
 
-#### [Watch the 30 Free Videos!](https://egghead.io/series/getting-started-with-redux) 
+#### [Watch the 30 Free Videos!](https://egghead.io/series/getting-started-with-redux)
```

Comment:
I can revert this change if ya want. My text editor stripped out the extra space.